### PR TITLE
UWM-RAR: Editor interface updates

### PR DESCRIPTION
**Pull request by sodacrackers**
UWM-RAR: Content updates to the RaR theme readme.md; grammar updates. Setting a min-height on body content, for layout builder form confirmation steps.

### CODE REVIEW

File: docroot/themes/custom/rar/readme.md [hunk 1]
```diff
@@ -11,15 +11,15 @@
 
 ### Getting Started
 
-Go to the theme directory (*/docroot/themes/custom/rar*), run `npm install` to get all the dependencies setup. After the installation of packages is completed, you can run `npm run build:dev` to compile code for a development environment. If you like to preview the files as they would be compiled for a production environment run `npm run build:prod`.
+Go to the theme directory (*/docroot/themes/custom/rar*), run `npm install` to download the dependencies above and project packages. After the npmm install, you can run `npm run build:dev` to compile code for a development environment. If you like to preview the files as if compiled for a production environment, run `npm run build:prod`.
```
Comment (blaquehat) [inline docroot/themes/custom/rar/readme.md:14]: Small spelling mistake `npmm` should be `npm`.

File: docroot/themes/custom/rar/src/css/base.css [hunk 2]
```diff
@@ -7,6 +7,9 @@ body {
   @apply text-gray-darkest;
 }
 
+main {
+  min-height: 500px;
```
Comment (blaquehat) [inline docroot/themes/custom/rar/src/css/base.css:11]: @sodacrackers, Is this meant for all breakpoints or a specific one?

**Review by blaquehat — CHANGES_REQUESTED** [review]

File: docroot/themes/custom/rar/readme.md (hunk 1 above)
Comment (sodacrackers) [reply docroot/themes/custom/rar/readme.md]: Typo fixed

File: docroot/themes/custom/rar/src/css/base.css (hunk 2 above)
Comment (sodacrackers) [reply docroot/themes/custom/rar/src/css/base.css:11]: All breakpoints; it seemed a reasonable safety since we do not have UX spec.

**Comment by blaquehat** [comment]
Closing due to improvements in https://github.com/uwmweb/uwmcms/pull/1944

**Pull request closed without merging**